src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -398,7 +399,7 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
                         addReplyToMessageContent(content, replyToEvent, {
                             permalinkCreator: this.props.permalinkCreator,
                             includeLegacyFallback: true,
-                            inThread: this.props.relation?.rel_type === RelationType.Thread,
+                            inThread: this.props.relation?.rel_type === THREAD_RELATION_TYPE.name,
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?